test/unit/generator/content/part2/part2.test.js

```diff
@@ -0,0 +1,114 @@
+const part2 = require('../../../../../app/generator/content/part2')
+let mockStatement
+
+describe('generate part 2', () => {
+  beforeEach(() => {
+    mockStatement = JSON.parse(JSON.stringify(require('../../../../mocks/statement-data')))
+  })
+
+  test('includes header', () => {
+    const result = part2(mockStatement)
+    expect(result[0].text).toBe('Part 2. Calculation')
+    expect(result[0].style).toBe('header2')
```

Comment:
split into 2 (soz to be that guy but yer use to it now :{P)

Reply:
Thought I'd got them all!  Darn